scripts/createEntrypoints.ts

```diff
@@ -1,23 +1,47 @@
 /* eslint @typescript-eslint/no-var-requires: 0 */
```

Comment:
Please add documentation of what this file contains, what its used for

Reply:
Added a description